--- a/src/qml/pages/Live.qml
+++ b/src/qml/pages/Live.qml
@@ -6,7 +6,6 @@
 
 import QtQuick
 import QtQuick.Layouts
-import QtQuick.Effects
 
 import org.kde.kirigami as Kirigami
 
@@ -35,23 +34,6 @@
         }
     ]
 
-    Kirigami.Icon {
-        anchors.centerIn: parent
-        width: Math.min(parent.height, Kirigami.Units.gridUnit * 16)
-        height: width
-
-        layer.enabled: installerAvailable && GraphicsInfo.api !== GraphicsInfo.Software
-        layer.effect: MultiEffect {
-            blurEnabled: true
-            blur: 1.0
-            blurMax: 10
-        }
-        opacity: installerAvailable ? 0.1 : 1
-
-        source: Welcome.Distro.logo
-        fallback: ""
-    }
-
     Welcome.ApplicationIcon {
         anchors.centerIn: parent
         application: Welcome.ApplicationInfo {
